doc/kci_rootfs.md

```diff
@@ -0,0 +1,157 @@
+
+                      {validate,list_configs,list_variants,build,upload} ...
+
+    optional arguments:
+
+      -h, --help            show this help message and exit
+
+      --yaml-configs YAML_CONFIGS
+
+                            Path to the YAML configs file
+    Commands:
+
+      {validate,list_configs,list_variants,build,upload}
+
+                            List of available commands
+
+        validate            Validate the rootfs YAML configuration
+
+        list_configs        List all rootfs config names
+
+        list_variants       List all rootfs variants
+
+        build               Build a rootfs image
+
+        upload              Upload a rootfs image
+    ```
+5. To list available rootfs configuration, you can use `list_configs` option.
+
+    ```
+    $ ./kci_rootfs list_configs
+
+    buster
+
+    buster-v4l2
+
+    buster-igt
+    ```
+   By default `kci_rootfs` reads entries from `rootfs-configs.yaml`. This file acts as a rootfs config file.
+
+6. Lets list all available rootfs images using `./kci_rootfs list_variants`. It should show existing
+rootfs name along with its architecture.
+
+    ```
+    buster amd64
+    buster arm64
+    ```
+
+    It also comes with couple of options `--config` and `--arch` to filter the results based on config name or arch type.
+
+    ```
+    $ ./kci_rootfs list_variants --config buster --arch i386
+    buster i386
+
+    $ ./kci_rootfs list_variants --config buster
+    buster amd64
+    buster arm64
+
+    $ ./kci_rootfs list_variants --arch amd64
+    buster amd64
+    buster-v4l2 amd64
+    buster-igt amd64
+    ```
+7. Now its time to re-build existing rootfs image using `kci_rootfs build`. It takes three arguments,
+`--config` refers to rootfs name which you want to build. `--data-path` points to debos files location.
+`--arch` refers to CPU arch you want to build. For example, to build a buster rootfs image for i386,
+you can run
+
+    ```
+    ./kci_rootfs build --config buster --data-path jenkins/debian/debos --arch i386
+
+    ```
+   depending on your network speed, this will take some time to complete.
+
+8. If build is successful you should see message like
+
+    ```
+    cd ${ROOTDIR} ; find -H  |  cpio -H newc -v -o | gzip -c - > ${ARTIFACTDIR}/buster/i386/rootfs.cpio.gz | ./build_info.json
+    cd ${ROOTDIR} ; find -H  |  cpio -H newc -v -o | gzip -c - > ${ARTIFACTDIR}/buster/i386/rootfs.cpio.gz | 79539 blocks
+    Poyouring off.
+    ==== Recipe done ====
+    ```
+    Finally newly built rootfs images can be found under the directory pointed by `--data-path`. In our case, its `jenkins/debian/debos/buster/i386/`
+
+    ```
+    $ ls jenkins/debian/debos/buster/i386/
+    build_info.json  full.rootfs.cpio.gz  full.rootfs.tar.xz  initrd.cpio.gz  rootfs.cpio.gz  rootfs.ext4.xz
+    ```
+
+### Create a new rootfs image
+
+Now you know how to build default kci_rootfs images. Lets look at how to add simple `stretch` rootfs image.
```

Comment:
`kci_rootfs` to be consistent with other places

Reply:
Also I don't think `stretch` is a good example as only Buster is used by KernelCI.  Maybe, showing how to create a new Buster variant would be more useful.

Reply:
ok, added `buster-example` rootfs creation as sample.